modules/core/src/matmul.cpp

```diff
@@ -1846,61 +1816,63 @@ transform_8u( const uchar* src, uchar* dst, const float* m, int len, int scn, in
 static void
 transform_16u( const ushort* src, ushort* dst, const float* m, int len, int scn, int dcn )
 {
-#if CV_SSE2
-    if( USE_SSE2 && scn == 3 && dcn == 3 )
+#if CV_SIMD128 && !defined(__aarch64__)
```

Comment:
@tomoaki0705 Just for clarification, why code is disabled for `__aarch64__`? performance regressions?

Reply:
Yes I disabled on ```__aarch64__``` for performance reason.
Auto optimization of compiler matched well for 16U on 64bit ARM

An example of measurement from ```Mat_Transform::Size_MatType::(1920x1080, 16UC3)``` on several ```__aarch64__``` platform.
The column "after" stands for universal intrinsic and "before" stands for pure C++ code with compiler optimization.
The x-factor had similar value on every size.

platform|before|after|x-factor
----|----|----|----
Jetson-TX1|64.104 ms|117.306 ms|0.55   
Jetson-TX2|47.393 ms|102.986 ms|0.46
ODROID-C2|145.647 ms|176.398 ms|0.83